django_freeradius/base/admin.py

```diff
@@ -198,7 +198,26 @@ def get_readonly_fields(self, request, obj=None):
 
 
 class AbstractRadiusProfileAdmin(TimeStampedEditableAdmin):
-    pass
+    list_display = ['name', 'get_daily_session_limit', 'get_daily_bandwidth_limit', 'get_max_all_time_limit',
```

Comment:
please list these vertically, experience taught me it's better (changing one item doesn't change the whole line and it's easier to track)

Reply:
:+1: